### PR TITLE
Add right hand sidebar for component docs

### PR DESCRIPTION
**Pull request by picklelo**

### CODE REVIEW

File: docs/library/forms/editor.md [hunk 1]
```diff
@@ -4,144 +4,3 @@ components:
 ---
 
 # Editor
-
-An HTML editor component based on [Suneditor](http://suneditor.com/sample/index.html).
```
Comment (masenf) [inline docs/library/forms/editor.md:8]: why is this doc removed?

**Review by masenf — COMMENTED** [review]
Working well for me

File: docs/library/forms/editor.md (hunk 1 above)
Comment (picklelo) [reply docs/library/forms/editor.md:8]: Was causing some issues, trying to bring it back

**Review by masenf — APPROVED** [review]

**Comment by tgberkeley** [comment]
Works well for me too

**Pull request merged**